cohttp-lwt-unix-bin/jbuild

```diff
@@ -0,0 +1,7 @@
+(jbuild_version 1)
+
+(executables
+ ((names (cohttp_curl_lwt cohttp_proxy_lwt cohttp_server_lwt))
+  (libraries (cohttp-lwt-unix cohttp_server cmdliner))
+  (package cohttp-lwt-unix-bin)
```

Comment:
Is it really worth having a separate OPAM package for the binaries? The cmdliner dependency is pretty minor and doesnt have a runtime impact on the cohttp_lwt_unix library. I'd suggest folding cohttp-lwt-unix-bin back into cohttp-lwt-unix